src/Unleash.php

```diff
@@ -14,4 +14,8 @@ public function isEnabled(string $featureName, ?Context $context = null, bool $d
     public function getVariant(string $featureName, ?Context $context = null, ?Variant $fallbackVariant = null): Variant;
 
     public function register(): bool;
+
+    public function getFeatures(): array;
+
+    public function getFeature(string $featureName);
```

Comment:
This doesn't have a return type, it should have one: https://github.com/Unleash/unleash-client-php/blob/main/src/Repository/UnleashRepository.php#L9